tests/integration/test_issue_1817.py

```diff
@@ -60,6 +63,7 @@ def test_sdist_for_project_with_native_extensions(tmpdir):
     assert_create_and_use_sdist_lock(tmpdir, "psutil==5.9.1", "import psutil")
 
 
+@pytest.mark.skipif(sys.version_info[:2] < (3, 6), reason="PyYAML 6.0.1 requires Python >= 3.6")
 def test_sdist_for_project_with_pep517_build(tmpdir):
     # type: (Any) -> None
-    assert_create_and_use_sdist_lock(tmpdir, "PyYAML==5.4.1", "import yaml")
+    assert_create_and_use_sdist_lock(tmpdir, "PyYAML==6.0.1", "import yaml")
```

Comment:
The 6.0.1 bump is a bitrot break forced by this whole hullabaloo:
+ https://github.com/yaml/pyyaml/issues/728
+ https://github.com/yaml/pyyaml/pull/726